pages/15.contributing/02.developer/01.Code/02.Pull requests/docs.md

```diff
@@ -317,7 +317,9 @@ Mautic uses [PHPUnit][php-unit] and [Selenium][selenium] as the suite of testing
 
 #### PHPUnit
 
-Before executing unit tests, copy the `.env.dist` file to `.env` then update to reflect your local environment configuration.
+**PHPUnit version:** Mautic 4.x requires [PHPUnit v5.7](https://phpunit.de/manual/5.7/en/installation.html), and Mautic 5.x requires [PHPUnit v9+](https://phpunit.de/announcements/phpunit-9.html). Make sure to install the right version of PHPUnit for the version of Mautic you are testing.
```

Comment:
Could you please make these into link macros, as used in the rest of the doc and listed below at the bottom of the page? Thanks!